### DISCUSSION TITLE
丹佛夜影 | /cn/2022/07/denver-shadow/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#1447] [Comments]
https://yihui.org/cn/2022/07/denver-shadow/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
啊？……你要是右键看一眼本页面的 HTML 源码，会看到我在开头的一段背景注释，里面就提到了《闪灵》。我没看过电影，也没看过小说，只是去年看史蒂芬金的《写作这回事》的时候听他自己提了一下，说是写他自己的酗酒心理，我也不知道里面的情节是什么样的，没想到它的地点也是在科罗拉多，这也太巧合了。

这是我第一次尝试写小说。跟写诗一样，我以前从没想过我会做这样的尝试。现在我感觉不管什么文学体裁，只要有基本的文字基础在，到了一定的时机，这些体裁就会从笔端自动冒出来，不必去刻意学习谁（鲁迅给青年的写作建议就是如果想写小说，最好别看什么学写小说的指导书籍）。文中的四个对象对应四种让我感到长久痛苦的情绪和心理，但我一直无法战胜它们，所以我想在小说中先战胜它们，算是给自己打打气，另外也是如我在最后一篇《河神》中所写，我要把它们从我身体里吐出来整理一下。

小说里面的很多对象和事件都是真实存在的，包括我有天晚上在客厅关灯时确实被自己的影子吓了一跳、孤零零的木屋和云杉、瓷熊、断鸭、烫胳膊、削手、坏锅炉也全都是真的。当然我也像揉面一样把它们重新揉了揉，以便托物言心。里面的确融入了我的一些童年生活经验。一般人可能最不熟悉的是我提到的桐油。我想到它的味道是因为小时候我爹经常用桐油浸渔网，所以我特别熟悉那味道，就有点像锅里或灶台上很久没清洗过的油渍味道。

谢谢你推荐的电影和书，我对这种题材的兴趣其实不算很大。我对重口不是很介意，但也不太喜欢。这篇里我是特意避免了使用重口的手段，只是把精力主要放在烘托气氛上，气氛出来就够了。

写完这篇我估计将来我不一定还能写出第二篇来，因为我的主要负面情绪都已经倾吐完了。将来要是再写，应该就会是积极明亮的色彩了。

> ——原帖发布于 2022-07-23 16:05:25

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
烫伤的颜色是记忆偏差，只记住了它最后的颜色，忘了刚开始的颜色。毕竟是新司机，细节写多了总会出现纰漏。

勇气确实是很重要，也很难攒够。

> ——原帖发布于 2022-07-24 18:38:44

**Reply by yihui** [reply]
这种颜色出入的暗示也有点太微妙了吧，估计绝大多数人都不会察觉的。

我现在有点倾向于相信共情更多的是偶然机缘使然，更少的是能力。偶然的意思是碰巧有共同的经历（比如小说中提到的轻生念头），自然而然就理解了。

小说到明年估计我也写不了几篇，说实话这篇写得我自己都有点喘，觉得自己还不是这块料。接下来我随意试试看吧，按鲁迅说的，写不出来就不硬写。

> ——原帖发布于 2022-08-09 13:41:33

[a comment by an automated account is not shown]

**Comment by hui-sheen** [comment]
我利用等程序运行的时间来随便读篇文章，怎料却遇到如此一篇宏篇巨制。中年人了，仿佛时间少到连读这样的文章也不够了，归根结底还是心情急躁了吧。

**Reply by yihui** [reply]
我认为主要原因并非心情急躁，而是时代驯化。我们被驯化得习惯了读零碎的句子、看短视频、吃快餐。就算遇到“长”文（几千字能算长文吗），也需要有人先划重点，以便后人只需要读那几句加粗的金句。遇到长视频，就用倍速看。

当然，我也并没有太悲观。短有短的好处和用处，但挑食总归不是个很好的习惯。不管时代怎样驯化，我相信一定会有人能坚持看长文的。时代淘洗的结果只是两类人沉浮地更分明而已。

时间跟金钱一样，说不够，那永远都不会够；说够，当下就够了。就看怎么花了。
